src/backend/distributed/citus--6.0-9--6.0-10.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE citus.pg_dist_transaction (
```

Comment:
We usually put a comment with the file name on top, which helps if you're reading the concatenated files.